src/graphic/Fast3D/gfx_pc.h

```diff
@@ -259,5 +273,8 @@ int32_t gfx_check_image_signature(const char* imgData);
 void gfx_register_blended_texture(const char* name, uint8_t* mask, uint8_t* replacement = nullptr);
 void gfx_unregister_blended_texture(const char* name);
 const char* GfxGetOpcodeName(int8_t opcode);
-
 #endif
+
+#ifdef __cplusplus
```

Comment:
This header is already c++ only. This guard is not need.

Also, put this function inside the `#endif` above. That is the end of the header guard and should always be the last line of the file.